docs/docs/overview/quickstart/get-started-with-kotlin.md

```diff
@@ -0,0 +1,117 @@
+- [Configure a Notification Channel Provider.](#set-up-a-channel-provider)
+- [Set up a Notification Workflow.](#create-a-notification-workflow)
+- Create Subscribers and update a Subscriber's information.
+- Send your first Notification to a subscriber.
+- Send Notifications via Topics.
+
+Let's get started! :muscle:
+
+## Requirements
+
+To be able to use this Quickstart, you'll need to have the following:
+
+- A Novu account, if you don't have one yet, [sign up](https://web.novu.co) for free
+- A working Kotlin project running version 1.6.0+ and Kotlin compiler 1.8.0+
+
+The completed code for this quick start will be available on GitHub soon. Check it out [here](https://github.com/novuhq)
+
+## Install and Set up the Novu Kotlin SDK in your project
+
+First, you need to import the library into your application.
+If you use Maven, add the dependency to the `pom.xml` file
+```xml
+<dependencies>
+  ...
+  <dependency>
+    <groupId>io.github.crashiv</groupId>
+    <artifactId>novu-kotlin</artifactId>
+    <version>0.1.1-SNAPSHOT</version>
+  </dependency>
+</dependencies>
+```
+Then add the repository
+```xml
+<repositories>
+    ...
+    <repository>
+        <url>https://s01.oss.sonatype.org/content/repositories/snapshots/</url>
+    </repository>
+</repositories>
+```
+
+If you use Gradle, add the dependency to the `build.gradle` file
+```groovy
+implementation 'io.github.crashiv:novu-kotlin:0.1.1-SNAPSHOT' //Groovy
+
+implementation ("io.github.crashiv:novu-kotlin:0.1.1-SNAPSHOT") //Kotlin
+```
+Then add the repository
+```groovy
+repositories {
+    ...
+    maven { url "https://s01.oss.sonatype.org/content/repositories/releases/" }
+}
+```
+Sync your project, and you should have the artefacts downloaded. Once that is done successfully, head over to the [web portal](https://web.novu.co) for the next steps
+
+## Set up a Channel Provider
+
+A channel provider is a service that provides one or more notification functionality such as sending an email, SMS, push notification etc. Our [integration store](https://web.novu.co/integrations) includes four channels: Email, SMS, Chat, and Push. These channels have multiple providers associated with them.
+
+| Channel | Providers                                                           |
+| ------- | ------------------------------------------------------------------- |
+| Email   | MailGun, Mandrill, MailJet, Amazon SES, Sendgrid, Postmark, Netcore |
+| SMS     | Twilio, Amazon SNS, Plivo, SMS, SMSCentral, Kannel, Infobip, Termii |
+| Chat    | Mattermost, Slack, Microsoft Teams, Discord                         |
+| Push    | FCM, APNS, Expo                                                     |
+
+Only one provider can be **active** per **channel**. Connect any of your favorite providers to get started. The email channel comes with Novu's email provider, which is active by default and includes 300 credits.
```

Comment:
I would make the Only one provider can be **active** per **channel** for <v0.16 as as we are looking to change this on v0.17.

Reply:
I think this works for now @Cliftonz. We'll update this across all quickstart guides once we release v0.17